webapps/ui/cockpit/src/modules/components/processInstance/CancelProcessInstance/CancelProcessInstance.js

```diff
@@ -37,10 +38,9 @@ const STATUS = Object.freeze({
   NOT_FOUND: 6
 });
 
-export default function CancelProcessInstance({ processInstanceId }) {
+function CancelProcessInstance({ id, definitionId }) {
```

Comment:
Note: Only works when initialized, as mentioned in the top comment 2. Can be worked around by initializing the context with `{}` instead of `null`
```suggestion
function CancelProcessInstance({ processInstance }) {
  const {id, definitionId} = processInstance;
```

Reply:
👍